tests/PySys/apt_plugin/apt_sequence_remove_with_version/run.py

```diff
@@ -27,20 +28,23 @@ class AptPluginRemoveWithVersion(AptPlugin):
     def setup(self):
         super().setup()
 
+        output = subprocess.check_output(["/usr/bin/apt-cache", "madison", "rolldice"])
```

Comment:
Before merging please make sure that madison is already on Rinas rpi. I believe it should be already there